packages/create-ship-app/src/scripts/cleanup.sh

```diff
@@ -6,6 +6,8 @@ api_dir="$1"
 api_type="$2"
 db_type="$3"
 
+rm $api_dir/src/ApiStarter.sln
```

Comment:
Shouldn't it be put after `if [ "$api_type" == ".NET" ]; then` line?

Btw, the same is for PR #149

Reply:
Yes, indeed.